iptables/patches/ip6tables.patch

```diff
@@ -0,0 +1,17 @@
+--- /usr/local/osquery/include/linux/netfilter_ipv6/ip6_tables.h
```

Comment:
this is purely a question on my end, but what exactly is going on with this patch file? how does it integrate with everything else (like the build system), if at all?

Reply:
I made this patch because of some weird behavior I was seeing across builds: clang would usually be smart enough to realize that the commented out inline function was unused code and eliminate it, but occasionally *wouldn't* and would fail the build due to the `void*` arithmetic in it. If the build succeeded for you without needing the patch, I'll just go ahead and remove it 😄

Reply:
Yeah, I did not do anything with this file and was able to build and run 🤷‍♂️